pkg/base/helm_test.go

```diff
@@ -90,11 +90,11 @@ func Test_RenderHelm(t *testing.T) {
 						},
 						{
 							Path:    "templates/deploy-1.yaml",
-							Content: []byte("file: 1\nkind: Deployment\nmetadata:\n  name: deploy-1\n  namespace: test-one"),
+							Content: []byte("apiVersion: v1\nkind: Deployment\nmetadata:\n  name: deploy-1\n  namespace: test-one"),
```

Comment:
deployments are `apps/v1` not `v1` i think :)

Reply:
hah fair enough 😄